tests/flytekit/unit/deck/test_deck.py

```diff
@@ -23,12 +25,30 @@ def test_deck():
     _output_deck("test_task", ctx.user_space_params)
 
 
+def test_time_line_deck():
+    time_info = dict(
+        Name="foo",
+        Start=datetime.datetime.utcnow(),
+        Finish=datetime.datetime.utcnow() + datetime.timedelta(microseconds=1000),
+        WallTime=1.0,
+        ProcessTime=1.0,
+    )
+    ctx = FlyteContextManager.current_context()
+    ctx.user_space_params._decks = []
+    time_line_deck = ctx.user_space_params.time_line_deck
+    time_line_deck.append_time_info(time_info)
+    assert time_line_deck.name == "time line"
+    assert len(time_line_deck.time_info) == 1
+    assert time_line_deck.time_info[0] == time_info
+    assert len(ctx.user_space_params.decks) == 1
```

Comment:
Unlike other decks, time_line_deck will the conversion of data to HTML is delayed until the html property is accessed since using partial data to render timeline graph is meaningless.


So, the time_line_deck.html file makes use of both GanttChartRenderer and TableRenderer for visualization. To enable these renderers, the 'flytekit-deck-standard' pluginneed be installed. However, as this plugin is not included in the ci testing environment, `time_line_deck.html` is not incorporated in this test.